src/main/java/org/cryptomator/common/settings/AbstractInvalidSetting.java

```diff
@@ -0,0 +1,23 @@
+package org.cryptomator.common.settings;
+
+import java.net.URI;
+
+public interface AbstractInvalidSetting {
+
+	/**
+	 * Returns the corresponding issue URI of this setting.<br>
+	 * The issue URI usually resolves to a page on the
+	 * <a href="https://github.com/cryptomator/cryptomator/issues">Cryptomator Bugtracker.</a>
+	 *
+	 * @return the issue URI or {@code null} if none is provided.
+	 */
+	URI getIssueURI();
```

Comment:
I need more context to understand what you're attempting to do here. How is pointing the user to some issue helping to reset to default values when `settings.json` contains garbage?